crates/nu-cli/src/completions/completer.rs

```diff
@@ -113,6 +193,11 @@ impl NuCompleter {
 
                         // Flags completion
                         if prefix.starts_with(b"-") {
+                            // Check if external completer
+                            if let Some(decl_id) = config.external_completer {
```

Comment:
The `decl_id`s should be called `block_id`s. It's a bit pedantic but it might be confusing if someone reads the code later.